core/integration/suite_test.go

```diff
@@ -64,7 +64,8 @@ func BenchMiddleware() []testctx.Middleware[*testing.B] {
 
 func connect(ctx context.Context, t testing.TB, opts ...dagger.ClientOpt) *dagger.Client {
 	opts = append([]dagger.ClientOpt{
-		dagger.WithLogOutput(testutil.NewTWriter(t)),
+		// FIXME: test spans are easier to read in the TUI when this is silenced
+		dagger.WithLogOutput(io.Discard),
```

Comment:
I could believe this is better, I find myself reading the plain output a lot but it might just be because it's there and I see it first. Commenting anyway in case you intended to undo this and forgot